### PR TITLE
Updated documentation formatting.

### PR DESCRIPTION
**Pull request by vovamarch**
Use doxygen 1.9.x to verify documentation build locally.

### CODE REVIEW

**Comment by vovamarch** [comment]
Actually only the last commit is relevant, so hope  you can merge without the history for others. Otherwise I can try to create a proper PR.

File: documentation/IoT_Socket.dxy [hunk 1]
```diff
@@ -1156,14 +1289,16 @@ HTML_EXTRA_STYLESHEET  = ./Doxygen_Templates/drv.css
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
 HTML_EXTRA_FILES       = ./Doxygen_Templates/tabs.css \
-                         ./Doxygen_Templates/tab_topnav.png \
                          ./Doxygen_Templates/search.css \
-                         ./Doxygen_Templates/printComponentTabs.js
+                         ./Doxygen_Templates/navtree.css \
+                         ./Doxygen_Templates/tab_b.png \
```
Comment (RobertRostohar) [inline documentation/IoT_Socket.dxy:1294]: File "tab_b.png" is missing.

File: documentation/IoT_Socket.dxy [hunk 2]
```diff
@@ -1156,14 +1289,16 @@ HTML_EXTRA_STYLESHEET  = ./Doxygen_Templates/drv.css
...
+                         ./Doxygen_Templates/printComponentTabs.js \
+                         ./Doxygen_Templates/navtree.js \
```
Comment (RobertRostohar) [inline documentation/IoT_Socket.dxy:1296]: File "navtree.js" is missing.

**Review by RobertRostohar — CHANGES_REQUESTED** [review]
In general looks ok but some files seem to be missing:
error: Extra file './Doxygen_Templates/tab_b.png' specified in HTML_EXTRA_FILES does not exist!
error: Extra file './Doxygen_Templates/navtree.js' specified in HTML_EXTRA_FILES does not exist!

File: documentation/IoT_Socket.dxy (hunk 1 above)
Comment (vovamarch) [reply documentation/IoT_Socket.dxy:1294]: Added the missing file.

File: documentation/IoT_Socket.dxy (hunk 2 above)
Comment (vovamarch) [reply documentation/IoT_Socket.dxy:1296]: Added the missing file.

**Review by RobertRostohar — APPROVED** [review]
Looks good now.

**Comment by RobertRostohar** [comment]
This PR has been rebased and merged into 'develop' branch (0984faf)

**Pull request closed without merging**